### PR TITLE
Include the course/module's instance in webhook

### LINKED ISSUE #80: Include edx_instance in CCXCon webhooks

### PR DESCRIPTION
**Pull request by justinabrahms**
Fixes #80

### CODE REVIEW

File: courses/models_test.py [hunk 1]
```diff
@@ -25,8 +25,10 @@ def test_towebhook(self):
         test to_webhook implementation returns valid json object
         """
         course = CourseFactory.build()
-        json.dumps(course.to_webhook())
-        ex_pk = course.to_webhook()['external_pk']
+        out = course.to_webhook()
+        json.dumps(out)
```
Comment (noisecapella) [inline courses/models_test.py]: This looks like a typo

Comment (noisecapella) [inline courses/models_test.py]: If it's here to verify that JSON serialization doesn't cause an exception, can you add a comment saying that?

Comment (justinabrahms) [inline courses/models_test.py]: Updated with comment. This is exactly what it was doing.

**Comment by giocalitri** [comment]
this is good to be merged :+1:

**Pull request merged**